SonarQube.TeamBuild.PreProcessor/Interfaces/IAnalyzerProvider.cs

```diff
@@ -7,6 +7,7 @@
 
 using SonarQube.Common;
 using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.PreProcessor.Roslyn;
```

Comment:
PropertiesWriter comment resolved with "no change required" - we write "sonar.projectKey" specifically in the project properties file because that changes on a component-to-component basis.

"sonar.branch" is correctly written globally when set.